.github/workflows/release-embedding-sdk.yml

```diff
@@ -84,14 +139,10 @@ jobs:
         with:
           m2-cache-key: "release-sdk"
 
-      - name: Update readme
-        run: |
-          ./bin/embedding-sdk/release_utils.bash update_readme ${{ inputs.sdk_version }}
```

Comment:
We have stopped doing this for a while, this is the step to update the readme to point to correct metabase version number, but now we link to prebuilt image and jar in the readme.